keyboards/ktec/ergodone/ergodox_compat.h

```diff
@@ -51,19 +51,25 @@ static inline void ergodox_right_led_off(uint8_t l) {
              break;
         }
 }
-static inline void ergodox_board_led_off(void) { setPinOutput(D5); writePin(D5, !ERGODOX_BOARD_LED_ON_STATE); }
-static inline void ergodox_board_led_on(void) { setPinOutput(D5); writePin(D5, ERGODOX_BOARD_LED_ON_STATE); }
+static inline void ergodox_rx_led_off(void) { setPinOutput(B0); writePin(B0, !ERGODOX_BOARD_LED_ON_STATE); }
```

Comment:
this breaks compatibility with ergodox led naming, which ... is the point of this file.